dev/breeze/src/airflow_breeze/global_constants.py

```diff
@@ -44,7 +44,7 @@
 ALLOWED_PYTHON_MAJOR_MINOR_VERSIONS = ["3.8", "3.9", "3.10", "3.11"]
 DEFAULT_PYTHON_MAJOR_MINOR_VERSION = ALLOWED_PYTHON_MAJOR_MINOR_VERSIONS[0]
 ALLOWED_ARCHITECTURES = [Architecture.X86_64, Architecture.ARM]
-ALLOWED_BACKENDS = ["sqlite", "mysql", "postgres", "mssql"]
+ALLOWED_BACKENDS = ["sqlite", "mysql", "postgres", "mssql", "none"]
```

Comment:
Maybe a comment to explain the `none` value? It might not be straightforward

Reply:
Yeah. Good idea.